src/meltano/core/utils/__init__.py

```diff
@@ -14,6 +14,7 @@
 from collections import OrderedDict
 from copy import deepcopy
 from datetime import date, datetime, time
+from distutils.util import strtobool
```

Comment:
Using this function from `distutils` feels a bit hacky. Let me know what you folks think.

Reply:
I'd rather we just re-implement it in `meltano.core.utils`.

Reply:
Its from `distutils` which is probably better than maintaining our own. I like how you have wrapped it though, so we can change the implementation in future if needed 👍

Reply:
@WillDaSilva fair enough: 8238fe9b67585f193c6eadd79de6714db900ddf7